app/views/va_facilities/_facility_created_practice_search.html.erb

```diff
@@ -17,7 +17,7 @@
           <label for="dm-created-practice-search-field" class="usa-label usa-sr-only">
             Search
           </label>
-          <input class="usa-input" id="dm-created-practice-search-field" autocomplete="off" type="search" name="created-practice-search" data-search="">
+          <input class="usa-input" id="dm-created-practice-search-field" title="Type at least three characters to search." autocomplete="off" type="search" name="search" data-search="">
```

Comment:
I don't think we need this `title` since technically they can search even with less than 3 characters

Reply:
Removed it.